PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx

```diff
@@ -1805,26 +1815,6 @@ Int_t AliAnalysisTaskMaterialHistos::CountTracks0814(){
 
   return fNumberOfESDTracks;
 }
-
-//________________________________________________________________________
-void AliAnalysisTaskMaterialHistos::DoSelectBCNumbers(){
```

Comment:
You could also have made this function return a boolean instead of hardcoding it in UserExec.